CONTRIBUTING.md

```diff
@@ -2,48 +2,72 @@
 
 Thank you for your interest in contributing to Okta's Open Source Projects! Before submitting a PR, please take a moment to read over our [Contributer License Agreement](https://developer.okta.com/cla/). In certain cases, we ask that you [sign a CLA](https://developer.okta.com/sites/all/themes/developer/pdf/okta_individual_contributor_license_agreement_2016-11.pdf) before we merge your pull request.
 
-## Style
+- [Commit Message Guidelines](#commit)
+- [Running Tests locally](#running_tests)
```

Comment:
Once rebasing, this link is not attached to our [README](/okta/okta-oidc-js#testing)